groupie/src/main/java/com/genius/groupie/ExpandableGroup.java

```diff
@@ -64,4 +64,71 @@ public void onToggleExpanded() {
             notifyItemRangeInserted(oldSize, newSize - oldSize);
         }
     }
+    
+    private boolean dispatchChildChanges(Group group) {
```

Comment:
That could definitely work as well. I don't have a strong feeling which I prefer